.circleci/test-targets.txt

```diff
@@ -5,6 +5,7 @@
 //stratum/hal/bin/bcm/standalone/...
 //stratum/hal/config/...
 //stratum/hal/lib/barefoot:bf_chassis_manager_test
+//stratum/hal/lib/barefoot:bf_pipeline_utils_test
```

Comment:
```suggestion
//stratum/hal/lib/barefoot/...
```

Reply:
Any reason not to include this? These are the only two tests and it will help someone from forgetting to add tests to CI if more are added in the future.